recipe/meta.yaml

```
@@ -33,28 +33,27 @@ requirements:
     - entrypoints >=0.3
     - jinja2 >=3
     - jsonschema >=3.2.0,<4.0
-    - jupyter-resource-usage >=0.5.1
     - jupyter_client >=6.1.7
+# https://github.com/elyra-ai/elyra/issues/3069    
     - jupyter_core >=4.11.2,<5
```

Let's remove the comment and the cap.  There's no reason to penalize all Python versions and 3.7 doesn't contain a `jupyter_core == 5` release anyway.
```suggestion 
    - jupyter_core >=4.11.2
```

---

See also https://github.com/elyra-ai/elyra/pull/3071 as it applies a conditional cap to `nbclient` relative only to Python 3.7 (in addition to removing `jupyter_core`'s cap).  Hopefully there's a way to apply the same kind of conditional behavior here.

---

I don't think we will need to add this one, it just wont resolve - I believe to do any python version specific conditions - we would need to make the entire elyra package move away from noarch